pkg/utils/env/env.go

```diff
@@ -87,3 +89,36 @@ func WithDefaultDuration(key string, def time.Duration) time.Duration {
 	}
 	return parsedVal
 }
+
+func GetRevision() string {
```

Comment:
Is this copied from somewhere? If so, can we add a comment for where this was copied from?

Reply:
Yes, this was basically the code from changeset.Get() from knative. Couldn't find a better alternative so had to copy what is being done there. I will add the comment.